src/python/other-bots/chromium-tests-syncer/run.py

```diff
@@ -208,6 +209,39 @@ def create_gecko_tests_directory(tests_directory, gecko_checkout_subdirectory,
                            target_subdirectory)
 
 
+def create_fuzzilli_tests_directory(tests_directory):
+  """Create Fuzzilli tests directory from the autozilli GCS archives."""
+  logs.info('Syncing fuzzilli tests.')
+  fuzzilli_tests_directory = os.path.join(tests_directory, 'fuzzilli')
+  remote_archive_tmpl = 'gs://autozilli/autozilli-%d.tgz'
```

Comment:
Is this readable by anyone?

Reply:
See: https://issues.chromium.org/issues/362963886#comment21